SearchAPI/CMR/Health.py

```diff
@@ -12,5 +12,5 @@ def get_cmr_health():
         logging.debug(e)
         d = {'host': cfg['cmr_base'], 'error': {
             'display': 'ASF is experiencing errors loading data.  Please try again later.',
-            'raw': '{0}'.format(e)}}
+            'raw': f'{e}'}}
```

Comment:
I'm very much thinking about removing the CMR health check from our own health check. It often creates false alarms, and slows things down.